### PR TITLE
Align README with the Python one

### PR DESCRIPTION
**Pull request by Giom-V**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,28 +10,22 @@ generative models features and behaviors.
...
-To use the Gemini API, you'll need an API key. If you don't already have one, 
-create a key in Google AI Studio: https://aistudio.google.com/app/apikey.
-
-### Dart samples
-
-See the Dart sample apps at [samples/dart](samples/dart/),
-including some getting started instructions.
+1. Go to [Google AI Studio](https://aistudio.google.com/).
+2. Login with your Google account.
+3. [Create](https://aistudio.google.com/app/apikey) an API key. Note that in Europe the free tier is not available.
```
Comment (natebosch) [inline README.md:19]: I don't know for sure that this violates any rules in https://developers.google.com/style/link-text#write-link-text, but I worry about the single word "Create" as a link.

```suggestion
3. [Create an API key](https://aistudio.google.com/app/apikey). Note that in Europe the free tier is not available.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,6 @@
 # Google Generative AI SDK for Dart
 
-The Google Generative AI SDK for Dart allows developers to use state-of-the-art
-Large Language Models (LLMs) to build language applications.
-
-See the [API overview of Gemini][Gemini API] for more information about
-generative models features and behaviors.
+The Google AI Dart SDK is the easiest way for Dart developers to build with the Gemini API. The Gemini API gives you access to Gemini [models](https://ai.google.dev/models/gemini) created by [Google DeepMind](https://deepmind.google/technologies/gemini/#introduction). Gemini models are built from the ground up to be multimodal, so you can reason seamlessly across text, images, and code.
```
Comment (natebosch) [inline README.md:3]: Dart repos use 80 character formatting for markdown and other files outside of `.dart`. Would you prefer I reformat this in a followup commit? There is also an internal markdown formatter which should use 80 character lines by default you could try.

File: README.md [hunk 3]
```diff
@@ -14,28 +10,22 @@ generative models features and behaviors.
...
-create a key in Google AI Studio: https://aistudio.google.com/app/apikey.
-
-### Dart samples
-
-See the Dart sample apps at [samples/dart](samples/dart/),
-including some getting started instructions.
+1. Go to [Google AI Studio](https://aistudio.google.com/).
+2. Login with your Google account.
+3. [Create](https://aistudio.google.com/app/apikey) an API key. Note that in Europe the free tier is not available.
+4. Try a Dart SDK [example](samples/dart) or the [Flutter ample app](samples/flutter_app/).
```
Comment (natebosch) [inline README.md:20]:
```suggestion
4. Try a [Dart CLI sample](samples/dart/) or the [Flutter sample app](samples/flutter_app/).
```

**Review by natebosch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Giom-V) [reply README.md]: Good point, I'll update the other repos the same way.

File: README.md (hunk 2 above)
Comment (Giom-V) [reply README.md]: That's a good point. I was surprised that this rule was not respected in the READMEs. I couldn't use `mdformat` in Visual Studio but I managed to find a way. It should be OK now.

**Review by natebosch — APPROVED** [review]

**Pull request merged**